tests/functional/modules/test_zos_job_submit_func.py

```diff
@@ -255,10 +255,56 @@
 //
 """
 
-JCL_FULL_INPUT="""//HLQ0  JOB MSGLEVEL=(1,1),
+JCL_FULL_INPUT = """//HLQ0  JOB MSGLEVEL=(1,1),
 //  MSGCLASS=A,CLASS=A,NOTIFY=&SYSUID
 //STEP1 EXEC PGM=BPXBATCH,PARM='PGM /bin/sleep 5'"""
 
+C_SRC_INVALID_UTF8 = """#include <stdio.h>
+int main()
+{
+        unsigned char a=0x64;
```

Comment:
Thanks for backporting this test @rexemin.

Could we add Hex `[0x15,0x0D, 0x25, 0x0E,0x0F]`  which are part of [DBBs doc](https://www.ibm.com/docs/en/dbb/1.0?topic=migrating-data-sets-git#managing-non-roundtrippable-characters) and the minimum viable subset that should also be included as part of testing. 

I pulled this from my full suite;
- `unsigned char _22=0x15;`
- `unsigned char _14=0x0D;`
- `unsigned char _38=0x25;`
- `unsigned char _15=0x0E;`
- `unsigned char _16=0x0F;`
- printf("Hex 0x%X is character = %c . \n",_22,_22);
- printf("Hex 0x%X is character = %c . \n",_14,_14);
- printf("Hex 0x%X is character = %c . \n",_38,_38);
- printf("Hex 0x%X is character = %c . \n",_15,_15);
- printf("Hex 0x%X is character = %c . \n",_16,_16);

Reply:
Sounds good, you probably don't have to run the pipeline again, we can just hold this PR till we have a 1.2.5.6, I am getting an early build today, I was going to do some isolated testing of the non-printables with the C code through a playbook, I can maybe give it @richp405 when I get it so they can upload it to the shared DASD for the pipeline to run.

Reply:
Looks good, thank you @rexemin